site/app/libraries/database/DatabaseQueries.php

```diff
@@ -797,7 +797,7 @@ public function setAnnouncement($thread_id, $onOff) {
         $this->course_db->query("UPDATE threads SET pinned = ? WHERE id = ?", [$onOff, $thread_id]);
     }
 
-    public function addPinnedThread($user_id, $thread_id, $added) {
+    public function addBookmarkedThread($user_id, $thread_id, $added) {
```

Comment:
Since these functions are getting renamed, now would be a good time to add PHP type hints as well.

https://submitty.org/developer/coding_style_guide/php#type-declarations